Week10/app/src/test/java/com/example/marvelcharacters/di/LocalRepositoryTest.kt

```diff
@@ -0,0 +1,48 @@
+package com.example.marvelcharacters.di
+
+import android.app.Application
+import com.example.marvelcharacters.repository.local.CharacterDao
+import com.example.marvelcharacters.repository.local.CharacterRepository
+import com.example.marvelcharacters.repository.local.RoomRepository
+import junit.framework.Assert.assertNotNull
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.koin.android.ext.koin.androidContext
+import org.koin.core.context.startKoin
+import org.koin.core.context.stopKoin
+import org.koin.test.KoinTest
+import org.koin.test.inject
+import org.mockito.Mock
+import org.mockito.MockitoAnnotations
+
+class LocalRepositoryTest : KoinTest {
```

Comment:
I'm not sure if you need these tests at all. 
You already have `ModuleTest` class that checks if all dependencies can be satisfied. 
Now, here, you are basically checking the same thing. At it takes 5 seconds just to run these 2 simple tests, because it has to create an entire Koin dependency map. 
5 seconds is a very long time for a unit test to run